stdlib/platform/java/thread/ThreadHelper.wyv

```diff
@@ -0,0 +1,27 @@
+module def ThreadHelper(java : Java) 
+
+import java:wyvern.stdlib.support.ThreadWrapper.threadwrapper
+import platform.java.thread.Thread
+import platform.java.thread.Runnable
+import platform.java.thread.ThreadGroup
+
+def make (f : Runnable) : Thread 
+	threadwrapper.makeThread(f)
+
+def activeCount() : Int
+	threadwrapper.activeCount()
+
+def interrupted() : Boolean
+	threadwrapper.interrupted()
+
+def sleep(m : Int, n : Int) : Unit
+	threadwrapper.sleep(m, n)
+
+def currentThread() : Thread
+	threadwrapper.currentThread()
+
+def join(t : Thread) : Unit
```

Comment:
Is there a race condition?  is it possible for isAlive() to change before join(t) is called, and would that cause a problem?

Reply:
I don't think so. join(t) halts the execution of the current thread until t dies or is interrupted so it shouldn't move past the join() to the assert until t is dead.

Reply:
Just rewrapped threads so join() is called from the Java side so there shouldn't be a possible race with the recursive call.